README.md

```diff
@@ -1,22 +1,27 @@
-# rust-web3
+# web3
 
 Ethereum JSON-RPC multi-transport client.
 Rust implementation of Web3.js library.
 
-[![Build Status][travis-image]][travis-url]
+[![Build Status][travis-image]][travis-url] [![Crates.io](https://img.shields.io/crates/v/web3)](https://crates.io/crates/web3) [![https://tomusdrw.github.io/rust-web3][documentation-badge]][documentation-url] [![docs.rs][docs-rs-badge]][docs-rs-url]
+
 
 [travis-image]: https://travis-ci.org/tomusdrw/rust-web3.svg?branch=master
 [travis-url]: https://travis-ci.org/tomusdrw/rust-web3
+[docs-rs-badge]: https://docs.rs/web3/badge.svg
+[docs-rs-url]: https://docs.rs/web3
+[documentation-badge]: https://img.shields.io/badge/Documentation-tomusdrw.github.io%2Frust--web3-blue
+[documentation-url]: https://tomusdrw.github.io/rust-web3
 
-[Documentation](http://tomusdrw.github.io/rust-web3/index.html)
+Documentation: [tomusdrw.github.io/rust-web3][documentation-url] or [docs.rs/web3](docs-rs-url)
```

Comment:
github pages documentation is for the `master` branch, and I think due to random release cycle it's still useful.

```suggestion
Documentation: [master branch][documentation-url] or [crates.io](docs-rs-url)
```